wallet/src/main/scala/org/bitcoins/wallet/Wallet.scala

```diff
@@ -506,13 +506,22 @@ abstract class Wallet
             new RuntimeException(s"Unable to find transaction ${txId.hex}"))
       }
 
+      _ = require(
+        tx.inputs.exists(_.sequence < TransactionConstants.disableRBFSequence),
```

Comment:
nit: This might be a useful utility elsewhere. Might be worth moving to `TxUtil.isRBFEnabled()`?